src/pkg/message/credentials.go

```diff
@@ -60,43 +62,56 @@ func PrintCredentialTable(state *types.ZarfState, componentsToDeploy []types.Dep
 }
 
 // PrintComponentCredential displays credentials for a single component
-func PrintComponentCredential(state *types.ZarfState, componentName string) {
+func PrintComponentCredential(ctx context.Context, state *types.ZarfState, componentName string) {
+	l := logger.From(ctx)
 	switch strings.ToLower(componentName) {
 	case GitKey:
 		Notef("Git Server push password (username: %s):", state.GitServer.PushUsername)
+		l.Info("Git server push password", "username", state.GitServer.PushUsername)
 		fmt.Println(state.GitServer.PushPassword)
 	case GitReadKey:
 		Notef("Git Server (read-only) password (username: %s):", state.GitServer.PullUsername)
+		l.Info("Git server (read-only) password", "username", state.GitServer.PullUsername)
 		fmt.Println(state.GitServer.PullPassword)
 	case ArtifactKey:
 		Notef("Artifact Server token (username: %s):", state.ArtifactServer.PushUsername)
+		l.Info("artifact server token", "username", state.ArtifactServer.PushUsername)
 		fmt.Println(state.ArtifactServer.PushToken)
 	case RegistryKey:
 		Notef("Image Registry password (username: %s):", state.RegistryInfo.PushUsername)
+		l.Info("image registry password", "username", state.RegistryInfo.PushUsername)
 		fmt.Println(state.RegistryInfo.PushPassword)
 	case RegistryReadKey:
 		Notef("Image Registry (read-only) password (username: %s):", state.RegistryInfo.PullUsername)
+		l.Info("image registry (read-only) password", "username", state.RegistryInfo.PullUsername)
 		fmt.Println(state.RegistryInfo.PullPassword)
 	default:
 		Warn("Unknown component: " + componentName)
+		l.Warn("unknown component", "component", componentName)
 	}
 }
 
 // PrintCredentialUpdates displays credentials that will be updated
-func PrintCredentialUpdates(oldState *types.ZarfState, newState *types.ZarfState, services []string) {
+func PrintCredentialUpdates(ctx context.Context, oldState *types.ZarfState, newState *types.ZarfState, services []string) {
```

Comment:
Coupling message together with logger should be avoided. What if instead we copied `PrintCredentialUpdates()` over to a helper in `cmd/tools/zarf.go` and then swapped out all of the message calls for logger calls? Then the new presentation logic stays in the presentation layer (cmd) and we don't intertwine both libraries.